### DISCUSSION TITLE
uninstall help

### DISCUSSION THREAD

**Discussion by magicsmoke2** [DISTRHO/Cardinal#305] [General]
Hi all,
Can anyone advise me on how to uninstall the latest version from a Mac please?
Many thanks

**Comment by falkTX** [comment]
There is no uninstaller.
But Cardinal does not install any system services or stuff that is run on the background, so you can safely leave it alone.

If you really want to delete the copied files, they are mostly in /Library/Audio/Plug-Ins/, then in either Component, LV2, VST or VST3, depending on what you picked on the installer.

**Reply by magicsmoke2** [reply]
Many thanks for the reply.
So, when I ran the installer, are you saying that all it did was copy files into set locations?
Many thanks again.

**Reply by falkTX** [reply]
Yes. the binaries are unsigned, so this is the only way to have things running on macOS, as the installer does not copy files with quarantine attributes set.

**Reply by falkTX** [reply]
the code for this is here https://github.com/DISTRHO/Cardinal/blob/main/utils/create-macos-installer.sh
with its additional files here https://github.com/DISTRHO/Cardinal/tree/main/utils/macOS

this is all 100% opensource, you can see everything that the program is doing.
you can build it yourself too and modify things, assuming you know how (computer programming etc)
